### PR TITLE
sys-kernel/ck-sources: bumps @ LTS (4.9.x / 4.14.x) branches

### PR DESCRIPTION
**Pull request by kuzetsa**
QA boot tested using "make defconfig" and qemu:

---

```
[    1.171896] MuQSS CPU scheduler v0.150 by Con Kolivas.
[Mon, 06 Aug 2018 03:46:45 +0000] boot complete.
Linux (none) 4.9.117-ck #1 SMP Sun Aug 5 22:44:01 EDT 2018  
x86_64 x86_64 x86_64 GNU/Linux
[Mon, 06 Aug 2018 03:46:45 +0000] powering off.
[    1.773721] reboot: Power down
```

---

```
[    1.173322] MuQSS CPU scheduler v0.162 by Con Kolivas.
[Mon, 06 Aug 2018 03:46:48 +0000] boot complete.
Linux (none) 4.14.60-ck #1 SMP Sun Aug 5 22:43:54 EDT 2018  
x86_64 x86_64 x86_64 GNU/Linux
[Mon, 06 Aug 2018 03:46:48 +0000] powering off.
[    1.745237] reboot: Power down
```

---

### notice ###

Only contains the MuQSS scheduler, genpatches, and any  
needed bugfixes for the respective branch (for build or  
runtime, etc.)

This change will reduce the number of kernel.org / LKML  
backports which can (and will) cause conflicts. The most  
recent was a bugfix from 4.18 which was backported to  
4.9; 4.14; 4.17 branches.

https://github.com/torvalds/linux/commit/9960521c44a5d828f29636ceac0600603ecbddbf

^ this touches: drivers/bluetooth/hci_qca.c

This bugfix is compatibile with MuQSS (the core of ckolivas'  
patchset), but other patches in the full patchset were not  
being actively maintained by upstream. for example, the  
specific tweak(s) which were added here:

* http://ck.kolivas.org/patches/4.0/4.9/4.9-ck1/patches/0008-Change-all-schedule_timeout-with-msecs_to_jiffies-po.patch
* http://ck.kolivas.org/patches/4.0/4.14/4.14-ck1/patches/0007-Replace-all-schedule-timeout-1-with-schedule_min_hrt.patch
* http://ck.kolivas.org/patches/4.0/4.17/4.17-ck1/patches/0007-Replace-all-schedule-timeout-1-with-schedule_min_hrt.patch

An unrelated conflict in the 4.17 branch is preventing a  
release, but the 4.18 window will will close soon. during  
the release of the full 4.18-ck1 patchset, a review will  
be done to determine if future "latest stable ck-sources"  
branch will be MuQSS-only, or the full ck patchset.

TL;DR - today's sys-kernel/ck-sources is LTS-only.

(*not holding out for a 4.17-ck2 compatibility patch*)

### CODE REVIEW

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/3d469d0e30a...d069396c1fc. Thanks! Sorry for the delay.
